wgpu-core/src/swap_chain.rs

```diff
@@ -71,7 +72,7 @@ pub enum PresentMode {
 #[derive(Clone, Debug)]
```

Comment:
Should the `SwapChainDescriptor` and `PresentMode` move as well?

Reply:
Yeah most likely, this PR isn't comprehensive because I didn't want this branch to diverge too much. I'm just moving types across whenever I need to use them with the web-sys backend (https://github.com/gfx-rs/wgpu-rs/pull/193), so there will be some smaller PRs later